docs/excel/performance.md

```diff
@@ -71,11 +71,15 @@ _Where:_
 
 Please be aware that some of the “properties” under an object may have the same name as another object. For example, `format` is a property under range object, but `format` itself is an object as well. So, if you make a call such as `range.load("format")`, this is equivalent to `range.format.load()`, which is an empty load() call that can cause performance problems as outlined previously. To avoid this, your code should only load the “leaf nodes” in an object tree. 
 
-## Suspend calculation temporarily
+## Suspend Excel processes temporarily
```

Comment:
Would this read better as "Temporarily suspend Excel processes" ? Just a suggestion; take it or leave it :)

Reply:
I agree it sounds better, but I wanted to match the verb-first header patterns.